nmclicktests/spec.authentication.js

```diff
@@ -0,0 +1,39 @@
+const expect = require('expect.js');
+const testuser = { username: 'testuser', password: 'testpassword' }
+const environment = require('./environment')({
+  serverStartupOptions: ['--authentication', '--users.' + testuser.username + '=' + testuser.password],
+  showServerOutput: true
+});
+
+describe('test authentication', () => {
+  before('Environment init', () => {
+    return environment.init();
+  });
+
+  it('Open home screen should show authentication dialog', () => {
+    return environment.goto(environment.url)
+      .wait('[data-ta-container="login-page"]');
```

Comment:
I know I used to have a different position, but I'm wondering if we shouldn't just move a away from the data-ta-container stuff and just use css classes instead?